.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted library name to the issue description
+
+**Library(ies) impacted by this PR:**
```

Comment:
Should this be packages instead of libraries?

Reply:
Changed it.